packages/components/theme.ts

```diff
@@ -8,49 +8,32 @@ export const theme = {
         heading: 'IBM Plex Sans, sans-serif',
     },
     buttons: {
```

Comment:
The button text has currently no styles, use variants to style them. Just add like ``variant: 'text.buttons.0'`` with the variant you think it match best

You can find all variants inside theme.ts in the `wip_design_update` branch
https://github.com/curaOS/source/blob/af9c891fc8c8b64622de273081fabf844cca7e72/packages/components/theme.ts#L38-L74